pageserver/src/branches.rs

```diff
@@ -155,7 +155,7 @@ pub fn create_repo(
     // Load data into pageserver
     // TODO To implement zenith import we need to
     //      move data loading out of create_repo()
-    bootstrap_timeline(conf, tenantid, tli, &*repo)?;
```

Comment:
Just a few unrelated nits I wanted to fix along the way, personally not fond of implicit `&*` things.
Lmk if you like the old version more, I'll revert.